src/redux/reducer.type.ts

```diff
@@ -42,16 +47,45 @@ export interface StudyUpdatedEventDataHeader {
     userId?: string;
 }
 
-export interface StudyUpdatedEventData {
+// Payloads
+export interface DeletedEquipment {
+    equipmentId: string;
+    equipmentType: string;
+}
+
+export interface NetworkImpactsInfos {
+    impactedSubstationsIds: UUID[];
+    deletedEquipments: DeletedEquipment[];
+    collectionElementImpacts: string[];
+}
+
+// EventData
+interface StudyUpdatedEventData {
     headers: StudyUpdatedEventDataHeader;
-    payload: string | any; // TODO check this
+    payload: NetworkImpactsInfos;
 }
 
-export interface StudyUpdated {
-    force: 0 | 1;
-    eventData: StudyUpdatedEventData;
+interface StudyUpdatedEventDataUnknown {
+    headers: StudyUpdatedEventDataHeader;
+    payload: string;
 }
 
+// Notification types
+type StudyUpdatedStudy = {
+    type: UpdateTypes.STUDY;
+    eventData: StudyUpdatedEventData;
+};
+
+type StudyUpdatedUndefined = {
+    type: undefined;
+    eventData: StudyUpdatedEventDataUnknown;
+};
+
+// Redux state
+export type StudyUpdated = {
+    force: 0 | 1;
```

Comment:
Why a number?

Reply:
It's a code which existed before this PR